modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheAdapter.java

```diff
@@ -659,13 +659,15 @@ protected final String startInfo() {
     /**
      * Stops this cache. Child classes should override this method
      * to provide custom stop behavior.
+     *
+     * @param destroy Destroy data flag. Setting to <code>true</code> will remove all cache data.
      */
-    public void stop() {
+    public void stop(boolean destroy) {
```

Comment:
We can avoid this change. See my comment to `GridCacheProcessor.stopCache()` method.